### PR TITLE
schema and bash files added

### PR DESCRIPTION
**Pull request by danfh00**
Schema and bash commands for connecting and clearing the schema added

### CODE REVIEW

File: Database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS price_readings, subscriptions, users, products;
+
+CREATE TABLE products(
+    product_id SERIAL PRIMARY KEY,
```
Comment (A-Faris) [inline Database/schema.sql:4]: Why did you use serial? This is an older version. Try to use default or default(1,1)

File: Database/connect.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql --host $DB_HOST -U $DB_USER -p $DB_PORT $DB_NAME
```
Comment (A-Faris) [inline Database/connect.sh:3]:
```suggestion
psql --host $DB_HOST -U $DB_USER -p $DB_PORT $DB_NAME -f schema.sql
```
Should it be this to connect to the schema?

File: Database/schema.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    subscription_id SERIAL PRIMARY KEY,
+    user_id INTEGER NOT NULL REFERENCES users (user_id),
+    product_id INTEGER NOT NULL REFERENCES products (product_id),
+    price_threshold DECIMAL(10, 2)
+);
+
+CREATE TABLE price_readings (
+    reading_id SERIAL PRIMARY KEY,
+    product_id INTEGER NOT NULL REFERENCES products (product_id),
+    timestamp TIMESTAMP NOT NULL,
```
Comment (A-Faris) [inline Database/schema.sql]:
```suggestion
    timestamp TIMESTAMP(0) NOT NULL,
```
This should remove trailing 0s

**Review by A-Faris — CHANGES_REQUESTED** [review]
Please add an `query.sql` file and a `query.sh` to connect to that file so we can quickly run query commands

File: Database/schema.sql (hunk 3 above)
Comment (danfh00) [reply Database/schema.sql]: Thanks! Will add this

**Pull request closed without merging**